auslib/db.py

```diff
@@ -1245,53 +1302,23 @@ def getOptions(self, username, permission, transaction=None):
         else:
             raise ValueError('Permission "%s" doesn\'t exist' % permission)
 
-    def hasUrlPermission(self, username, url, method, urlOptions={}, transaction=None):
-        """Check if a user has access to an URL via a specific HTTP method.
-           GETs are always allowed, and admins can always access everything."""
+    def hasPermission(self, username, thing, action, product=None, transaction=None):
```

Comment:
Yeah. The new permission names ("build", "rule", "release_locale", etc.) got me thinking of them as "objects" that can do different things to ("actions"). "object" is a reserved word, so I went with "thing". I'm not particularly happy with the name either - would you prefer "permission"?